bridge/slack/slack.go

```diff
@@ -296,12 +296,14 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 		return msg.ID, err
 	}
 
-	messageParameters := b.prepareMessageParameters(&msg)
+	var messageOptions []slack.MsgOption
 
 	// Upload a file if it exists.
 	if msg.Extra != nil {
 		for _, rmsg := range helper.HandleExtra(&msg, b.General) {
-			_, _, err = b.rtm.PostMessage(channelInfo.ID, rmsg.Username+rmsg.Text, *messageParameters)
+			messageOptions = b.prepareMessageOptions(&rmsg)
+			rmsg.Text = rmsg.Username + rmsg.Text
+			_, err = b.postMessage(&rmsg, messageOptions, channelInfo)
```

Comment:
Moved to using postMessage here

Reply:
Ok, so I think this comment is now obsolete, right? The args are aligned, and there's no difference in calls to rtm vs rest